### PR TITLE
Update documentation and tutorial for 2.0

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

This PR updates both documentation and tutorial apps to reflect the new Core 2.0 dependencies.
Start/Final app:
1. Update dependencies to production 2.0.0 versions (will be broken until production versions are available)
2. Update logs to use Core Services Log
    * public LOG_TAG defined in MainApp and static imported into other tutorial app files where needed
3. **res/xml files updated to move the xml definition to the very top (above the generated license), which was causing build errors**
4. Remove jetifier from gradle.properties (assurance 2.0)

Edge Tutorial:
1. Replace all gitbook docs with updated doc links

API reference:
1. Add deprecation warning to `registerExtension` API

Main README:
1. Update dependencies to latest 2.0.0 version with link to latest extension versions
2. Replace outdated doc links
3. **Update mobile property instructions to refer to more comprehensive tutorial instructions**
4. Misc. doc cleanup

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,38 +5,36 @@
...
+> See the [current version list](https://developer.adobe.com/client-sdks/documentation/current-sdk-versions) for the latest extension versions to use.
 
 ### Development
 
 **Open the project**
 
 To open and run the project, open the `code/settings.gradle` file in Android Studio.
 
-**Data Collection mobile property prerequisites**
+#### **Adobe Experience Platform mobile property prerequisites**
```
Comment (addb) [inline README.md:30]: nit: Do we require bold here? since its already a header?

File: README.md [hunk 2]
```diff
@@ -5,38 +5,36 @@
 
 The Adobe Experience Platform Edge Network mobile extension allows you to send data to the Adobe  Edge Network from a mobile application. This extension allows you to implement Adobe Experience Cloud capabilities in a more robust way, serve multiple Adobe solutions though one network call, and simultaneously forward this information to the Adobe Experience Platform.
 
-The AEP Edge Network mobile extension is an extension for the [Adobe Experience Platform SDK](https://aep-sdks.gitbook.io) and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for retrieving the identities, such as ECID.
+The AEP Edge Network mobile extension is an extension for the [Adobe Experience Platform SDK](https://developer.adobe.com/client-sdks/documentation/) and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for handling identities, such as `ECID`.
```
Comment (addb) [inline README.md:8]: Let's move all links to one section on Readme and reference that section.

File: README.md [hunk 3]
```diff
@@ -5,10 +5,15 @@
...
 
-The AEP Edge Network mobile extension is an extension for the [Adobe Experience Platform SDK](https://developer.adobe.com/client-sdks/documentation/) and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for handling identities, such as `ECID`.
+The AEP Edge Network mobile extension is an extension for the Adobe Experience Platform SDK and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for handling identities, such as `ECID`.
 
-To learn more about this extension, read the [Adobe Experience Platform Edge Network](https://developer.adobe.com/client-sdks/documentation/edge-network/) documentation.
+## Documentation
+
+To learn more about this extension, see the [Adobe Experience Platform Edge Network](https://developer.adobe.com/client-sdks/documentation/edge-network/) documentation.
+
+Additional information about SDK usage and architecture can be found in the [Documentation](Documentation) directory.
```
Comment (addb) [inline README.md:15]: Can we reword to something on these lines:

Information regarding AEP Edge Network extension implementation, API usage and architecture can be found in [Documentation](Documentation) directory

To Learn more about AEP Edge Network extension along with all the other AEP Mobile extensions on the official [AEP Mobile documentation](https://developer.adobe.com/client-sdks/documentation/edge-network/).

File: README.md [hunk 4]
```diff
@@ -5,38 +5,41 @@
...
+## Getting started
 ### Installation
 
 Integrate the Edge Network extension into your app by including the following in your gradle file's `dependencies`:
 
 ```gradle
-implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
-implementation 'com.adobe.marketing.mobile:edge:1.+'
-implementation 'com.adobe.marketing.mobile:core:1.+'
+implementation 'com.adobe.marketing.mobile:core:2.0.0'
```
Comment (cacheung) [inline README.md:22]: We will point this to getting started page.

File: Documentation/api-reference.md [hunk 5]
```diff
@@ -70,6 +70,9 @@ Edge.getLocationHint(new AdobeCallbackWithError<String>() {
 
 Registers the Edge Network extension with the Mobile Core extension.
 
+> **Warning**  
+> Deprecated as of 2.0.0. Use the [MobileCore.registerExtensions API](https://developer.adobe.com/client-sdks/documentation/mobile-core/api-reference) instead.
```
Comment (cacheung) [inline Documentation/api-reference.md:74]: let change the link to https://github.com/adobe/aepsdk-core-android

File: README.md (hunk 4 above)
Comment (cacheung) [reply README.md:22]: Integrate the Edge Network extension into your app by including the following the [getting started guide](Documentation/getting-started.md).

Comment (cacheung) [reply README.md:22]: Are you going to update getting started to 2+ in this PR?
https://github.com/timkimadobe/aepsdk-edge-android/blob/update-tutorial-2.0/Documentation/getting-started.md#add-the-edge-network-extension-to-your-app

Comment (cacheung) [reply README.md:22]: You can wait for edgeIdentity doc PR reviewed then change.

**Review by cacheung — COMMENTED** [review]
getting-started.md still have two links pointing to
aep-sdks.gitbook doc

File: Documentation/Tutorials/EdgeTutorialAppStart/app/src/main/java/com/adobe/marketing/mobile/tutorial/MainApp.java [hunk 6]
```diff
@@ -12,25 +12,26 @@
...
 import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.services.Log;
+
+import java.util.Arrays;
 // Edge Tutorial - code section (1/2) */
 
 public class MainApp extends Application {
-    private static final String LOG_TAG = "Test Application";
+    public static final String LOG_TAG = "EdgeTutorialApp";
+    private static final String LOG_SOURCE = "Test Application";
```
Comment (emdobrin) [inline Documentation/Tutorials/EdgeTutorialAppStart/app/src/main/java/com/adobe/marketing/mobile/tutorial/MainApp.java:34]: maybe LOG_SOURCE = "MainApp" for this class?

File: Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/MainApp.java [hunk 7]
```diff
@@ -12,25 +12,26 @@
...
 import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.services.Log;
+
+import java.util.Arrays;
 // Edge Tutorial - code section (1/2) */
 
 public class MainApp extends Application {
-    private static final String LOG_TAG = "Test Application";
+    public static final String LOG_TAG = "EdgeTutorialApp";
+    private static final String LOG_SOURCE = "Test Application";
```
Comment (emdobrin) [inline Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/MainApp.java:34]: "MainApp"

File: Documentation/Tutorials/edge-send-event-tutorial.md [hunk 8]
```diff
@@ -364,7 +365,7 @@ Make sure to uncomment all sections within each file (the total will tell you ho
...
+Notice that the Lifecycle APIs rely on the developer to place them in the proper app lifecycle functions (as seen in `MainActivity.java` and `AssuranceActivity.java`); that is, there are built-in functions that are called by the operating system that give the app notices that it is about to enter an active state, or go into a background state, etc. A proper Lifecycle extension implementation requires that the developer places the API calls in the required Android lifecycle functions. See the full guide on [implementing Lifecycle](https://developer.adobe.com/client-sdks/documentation/mobile-core/lifecycle/).
```
Comment (emdobrin) [inline Documentation/Tutorials/edge-send-event-tutorial.md:368]: I think we can remove "See the full guide .."

File: README.md [hunk 9]
```diff
@@ -3,40 +3,43 @@
...
+The Adobe Experience Platform Edge Network mobile extension allows you to send data to the Adobe Edge Network from a mobile application. This extension allows you to implement Adobe Experience Cloud capabilities in a more robust way, serve multiple Adobe solutions though one network call, and simultaneously forward this information to the Adobe Experience Platform.
 
-The AEP Edge Network mobile extension is an extension for the [Adobe Experience Platform SDK](https://aep-sdks.gitbook.io) and requires the `AEPCore` and `AEPServices` extensions for event handling, as well as the `AEPEdgeIdentity` extension for retrieving the identities, such as ECID.
+The Edge Network mobile extension is an extension for the Adobe Experience Platform SDK and requires the Mobile Core and Services extensions for event handling, as well as the Identity for Edge Network extension for handling identities, such as `ECID`.
```
Comment (emdobrin) [inline README.md:8]: You can use `https://developer.adobe.com/client-sdks` here for Adobe Experience Platform Mobile SDK

File: README.md [hunk 10]
```diff
@@ -47,24 +50,19 @@ Configure a new Assurance session by setting the Base URL to `testapp://main` an
 $ adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=ADD_YOUR_SESSION_ID_HERE" com.adobe.marketing.edgetestapp
 ```
 
-Note: replace ADD_YOUR_SESSION_ID_HERE with your Assurance session identifier.
+Note: replace `ADD_YOUR_SESSION_ID_HERE` with your actual Assurance session identifier.
 
-Once the connection is established and the events list starts getting populated, you can filter the Edge extension events. See full list of available events [here](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension/validation).
+Once the connection is established and the events list starts getting populated, you can filter the Edge extension events. See the full list of available events [here](https://developer.adobe.com/client-sdks/documentation/edge-network/validation/).
```
Comment (emdobrin) [inline README.md:55]: For this README we should apply similar changes as https://github.com/adobe/aepsdk-edgeconsent-android/pull/60

**Review by emdobrin — COMMENTED** [review]
@timkimadobe your changes look great
left small cleanup comments, aside from the README that needs few more updates

File: Documentation/getting-started-test-app.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+1. In the test app, set your `ENVIRONMENT_FILE_ID` in `TestApplication.java`.
+2. Select the `app` runnable with the desired emulator and run the program.
+
+## Validation with Assurance
+
+Configure a new Assurance session by setting the Base URL to `testapp://main` and launch Assurance in the test app by running the following command in your terminal:
+
+```bash
+$ adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=ADD_YOUR_SESSION_ID_HERE" com.adobe.marketing.mobile.consenttestapp
```
Comment (cacheung) [inline Documentation/getting-started-test-app.md:25]: I am not sure if this command line does work or not, trying to testing.  But for Edge it should not be com.adobe.marketing.mobile.consenttestapp.

Comment (cacheung) [reply Documentation/getting-started-test-app.md:25]: Should be this one
adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=33803986-ae78-4679-975e-4bda17ca8b81" com.adobe.marketing.edgetestapp

File: README.md [hunk 12]
```diff
@@ -1,70 +1,53 @@
...
+| [Identity for Edge Network extension](https://github.com/adobe/aepsdk-edgeidentity-android)        | The Identity for Edge Network extension enables handling of user identity data from a mobile app when using the Experience Platform Mobile SDK and the Edge Network extension. |
+| [Adobe Experience Platform Assurance](https://github.com/adobe/aepsdk-assurance-android)           | The Assurance extension enables validation workflows for your Experience Platform Mobile SDK implementation. |
+| [Adobe Experience Platform Android sample app](https://github.com/adobe/aepsdk-sample-app-android) | Contains a fully implemented Android sample app using the Experience Platform Mobile SDK. |
 
 ## Documentation
 
-Additional documentation for usage and SDK architecture can be found under the [Documentation](Documentation) directory.
+Information about Adobe Experience Platform Edge Network's implementation, API usage, and architecture can be found in the [Documentation](Documentation) directory.
```
Comment (emdobrin) [inline README.md:48]: can you include Getting started with the test app in the Documentation/README.md contents table?

File: Documentation/getting-started.md [hunk 13]
```diff
@@ -1,89 +1,114 @@
...
+	}
+}
+```
 
+#### Kotlin
+```kotlin
+class MainApp : Application() {
 
-1. Add the Mobile Core, Identity for Edge Network, and Edge Network extensions to your project using the app's Gradle file:
+  private var ENVIRONMENT_FILE_ID: String = ""
```
Comment (emdobrin) [inline Documentation/getting-started.md:98]: nit: you can use ENVIRONMENT_FILE_ID: String = "YOUR_APP_ENVIRONMENT_ID" similar to java code

File: Documentation/getting-started.md [hunk 14]
```diff
@@ -1,89 +1,114 @@
...
+
+	@Override
+	public void onCreate() {
+		super.onCreate();
+
+		MobileCore.setApplication(this);
+		MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+
+		MobileCore.registerExtensions(
+			Arrays.asList(Consent.EXTENSION, Edge.EXTENSION, Identity.EXTENSION),
```
Comment (emdobrin) [inline Documentation/getting-started.md:87]: consent is not installed/imported, maybe we can remove from these two examples

File: Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java [hunk 15]
```diff
@@ -134,12 +137,12 @@ public void onClick(View view) {
                 ExperienceEvent event = new ExperienceEvent.Builder()
                         .setXdmSchema(eventData)
                         .build();
-                Log.d(LOG_TAG, "Sending event");
+                Log.debug(LOG_TAG, LOG_SOURCE, "Sending event");
```
Comment (addb) [inline Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java:140]: nit: "Sending ProductView Edge event"

File: Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java [hunk 16]
```diff
@@ -91,12 +93,13 @@ public void onClick(View view) {
...
+
                 Edge.sendEvent(event, new EdgeCallback() {
 
                     @Override
                     public void onComplete(final List<EdgeEventHandle> handles) {
-                        Log.d(LOG_TAG, "Edge event callback called");
+                        Log.debug(LOG_TAG, LOG_SOURCE, "Edge event callback called");
```
Comment (addb) [inline Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java:102]: nit: "Callback called for ProductAdd Edge event"

File: Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java [hunk 17]
```diff
@@ -91,12 +93,13 @@ public void onClick(View view) {
                 ExperienceEvent event = new ExperienceEvent.Builder()
                         .setXdmSchema(xdmData)
                         .build();
-                Log.d(LOG_TAG, "Sending event");
+                Log.debug(LOG_TAG, LOG_SOURCE, "Sending event");
```
Comment (addb) [inline Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java:96]: nit: "Sending ProductAdd Edge event"

File: Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java [hunk 18]
```diff
@@ -134,12 +137,12 @@ public void onClick(View view) {
...
                         .setXdmSchema(eventData)
                         .build();
-                Log.d(LOG_TAG, "Sending event");
+                Log.debug(LOG_TAG, LOG_SOURCE, "Sending event");
                 Edge.sendEvent(event, new EdgeCallback() {
 
                     @Override
                     public void onComplete(final List<EdgeEventHandle> handles) {
-                        Log.d(LOG_TAG, "Edge event callback called");
+                        Log.debug(LOG_TAG, LOG_SOURCE, "Edge event callback called");
```
Comment (addb) [inline Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/EdgeFragment.java:145]: nit: "Callback called for ProductAdd Edge event"

File: Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/MainActivity.java [hunk 19]
```diff
@@ -48,7 +50,7 @@ protected void onCreate(Bundle savedInstanceState) {
             // Enables deep linking to connect to Assurance.
             //* Edge Tutorial - code section (2/4)
             Assurance.startSession(data.toString());
-            MobileCore.log(LoggingMode.VERBOSE, LOG_TAG, "Deep link received " + data.toString());
+            Log.debug(LOG_TAG, LOG_SOURCE, "Deep link received: " + data.toString());
```
Comment (addb) [inline Documentation/Tutorials/EdgeTutorialAppFinal/app/src/main/java/com/adobe/marketing/mobile/tutorial/MainActivity.java:53]: nit: Deeplink for Assurance received ...

File: Documentation/api-reference.md [hunk 20]
```diff
@@ -70,6 +70,9 @@ Edge.getLocationHint(new AdobeCallbackWithError<String>() {
 
 Registers the Edge Network extension with the Mobile Core extension.
 
+> **Warning**  
+> Deprecated as of 2.0.0. Use the [MobileCore.registerExtensions API](https://github.com/adobe/aepsdk-core-android) instead.
```
Comment (addb) [inline Documentation/api-reference.md:74]: update url: https://github.com/adobe/aepsdk-core-android/blob/staging/Documentation/MobileCore/api-reference.md

Comment (emdobrin) [reply Documentation/api-reference.md:74]: let's use main branch for all link references:
https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/api-reference.md

**Review by addb — APPROVED** [review]
looks good!

**Pull request merged**